src/components/NewAnalysisModal.js

```diff
@@ -100,8 +262,11 @@ export const NewAnalysisModal = Utils.withDisplayName('NewAnalysisModal')(
         title: 'Select an application',
         titleStyles: _.merge(viewMode === undefined ? {} : { display: 'none' }, { margin: '1.5rem 0 0 1.5rem' }),
```

Comment:
Since merge does a deep merge, and styles are flat, we can avoid that usage.
```suggestion
      titleStyles: { margin: '1.5rem 0 0 1.5rem', display: !!viewMode ? undefined : 'none' },
```